_assets/js/netlify/makeTemplate.js

```diff
@@ -4,12 +4,17 @@ import formatPublishDates from './formatPublishDates';
 /** @jsx h */
 
 function makeTemplate(data) {
-  const { title, leadText, body, publishDate, updatedDate, print, pdf, relatedContent } = data;
+  const { previewLink, title, leadText, body, publishDate, updatedDate, print, pdf, relatedContent } = data;
   const pageTemplate = (
     <div
       id="crt-page--content"
       className="mobile-lg:grid-col tablet:grid-col-12 desktop:grid-col-8"
     >
+      {previewLink ? (
+        <div className="width-full padding-bottom-2 padding-top-2 border-bottom">
+          <a href={previewLink} target="_blank" className="margin-top-4 text-bold line-height-mono-6">Page preview link</a>
+        </div>
+      ) : null}
```

Comment:
is it worth adding a "Move to review to get a preview link" instead of hiding this completely?

(Users might be confused by its absence if they expect a link to exist / don't understand how/when the link is created)

Reply:
good call, added!